atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -278,10 +278,14 @@ def get_source_registry(workflow, fallback=NO_FALLBACK):
             return fallback
         raise
 
-    return {
+    regdict = {
         'uri': RegistryURI(source_registry['url']),
-        'insecure': source_registry.get('insecure', False)
+        'insecure': source_registry.get('insecure', False),
+        'secret': None
```

Comment:
Could we call this `dockercfg_path` to make it clearer what value to expect there?

Reply:
Yes